app/presenters/queries/content_item_presenter.rb

```diff
@@ -15,7 +15,10 @@ class ContentItemPresenter
         :state_history,
         :change_note,
         :links,
-      ] - [:state]).freeze # state appears as 'publication_state'
+      ] - [
+        :state, # state appears as 'publication_state'
+        :need_ids # has been deprecated
```

Comment:
Yeah... why not. I can't remember quite why I didn't do that initially.

I've updated the first commit to do this, rather than making this change here.